ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -251,6 +251,152 @@
+          "recipe": {
+            "contentType": "text/x-yaml",
+            "type": "kubernetes",
+            "content": "kind: List\nitems:\n - \n  apiVersion: v1\n  kind: Pod\n  metadata:\n   name: web\n   labels:\n    type: db\n  spec:\n   containers:\n    - \n     image: eclipse/centos_jdk8\n     name: dev\n     ports:\n      - \n       containerPort: 8080\n       protocol: TCP\n     resources:\n      limits:\n       memory: 512Mi\n    - \n     image: centos/mysql-57-centos7\n     name: mysql\n     ports:\n      - \n       name: mysql\n       containerPort: 3306\n       protocol: TCP\n     env:\n      - \n       name: MYSQL_USER\n       value: petclinic\n      - \n       name: MYSQL_ROOT_PASSWORD\n       value: password\n      - \n       name: MYSQL_PASSWORD\n       value: password\n      - \n       name: MYSQL_DATABASE\n       value: petclinic\n     resources:\n      limits:\n       memory: 256Mi\n - \n  apiVersion: v1\n  kind: Service\n  metadata:\n   name: db\n  spec:\n   selector:\n    type: db\n   ports:\n    - \n     name: mysql\n     port: 3306\n     protocol: TCP\n     targetPort: mysql\n"
+          },
+          "machines": {
+            "web/dev": {
+              "servers": {
+                "tomcat8": {
+                  "attributes": {},
+                  "protocol": "http",
+                  "port": "8080/tcp"
+                }
+              },
+              "volumes": {
+                "projects": {
+                  "path": "/projects"
+                }
+              },
+              "installers": [],
+              "env": {},
+              "attributes": {}
+            },
+            "web/mysql": {
+              "servers": {
+                "mysql": {
+                  "attributes": {},
+                  "port": "3306",
+                  "protocol": "tcp"
+                }
+              },
+              "volumes": {},
+              "installers": [],
+              "env": {},
+              "attributes": {}
+            }
+          }
+        }
+      },
+      "defaultEnv": "default",
+      "name": "default",
+      "projects": [],
+      "commands": [
+        {
+          "commandLine": "cd /projects && git clone https://github.com/che-samples/web-java-spring-petclinic.git && cd web-java-spring-petclinic && git checkout mysql",
+          "name": "clone petclinic sample project",
+          "attributes": {
+            "machineName": "web/dev",
+            "previewUrl": "",
+            "goal": "Run"
+          },
+          "type": "custom"
+        },
+        {
+          "commandLine": "/opt/rh/rh-maven33/enable && mvn clean install -f ${current.project.path}",
+          "name": "build",
+          "type": "mvn",
+          "attributes": {
+            "machineName": "web/dev",
+            "previewUrl": "",
+            "goal": "Build"
+          }
+        },
+        {
+          "commandLine": "cd ${current.project.path}/target/ && cp *.war ${TOMCAT_HOME}/webapps/ROOT.war",
```

Comment:
This command won't work with Theia, right?

Reply:
It will if user selects project in Files panel.